### DISCUSSION TITLE
not able to view time graph

### DISCUSSION THREAD

**Discussion by maheswarGR** [grafana/grafana-infinity-datasource#218] [Q&A]
@yesoreyeram Hi i tried same data which is given in https://sriramajeyam.com/grafana-infinity-datasource/wiki/csv
datasource : Infinity
choosed type : csv, inline, timeseries
![image](https://user-images.githubusercontent.com/91525476/147072392-1562af65-dfc5-4a4c-acec-8d5fbdbf99ef.png)
![image](https://user-images.githubusercontent.com/91525476/147072668-cca61228-0ba2-4afd-b8e2-212aacfcb3a6.png)

**Comment by yesoreyeram** [comment]
The data doesn't habe time field. If you look at tabke or other panels, you will see a data.
